.github/actions/push/action.yml

```diff
@@ -26,7 +26,7 @@ runs:
   using: "composite"
   steps:
     -
-      uses: actions/download-artifact@v2
+      uses: actions/download-artifact@v4
```

Comment:
Had to be upgraded because of issues with having so many artifacts in a single workflow now, and those issues are resolved in v4.